### PR TITLE
Rename orbit averaged omega22

### PR DESCRIPTION
**Pull request by md-arif-shaikh**
Addressing issue #189. Basically changes `orbit_averaged_omega22_at` -> `orbit_averaged_omega22_between`.

### CODE REVIEW

File: gw_eccentricity/eccDefinition.py [hunk 1]
```diff
@@ -1525,9 +1525,13 @@ def get_t_average_for_orbit_averaged_omega22(self):
         return [t_for_orbit_averaged_omega22,
                 sorted_idx_for_orbit_averaged_omega22]
 
-    def get_orbit_averaged_omega22_at_pericenters(self):
+    def get_orbit_averaged_omega22_between_pericenters(self):
         """Get orbital average of omega22 between two consecutive pericenters.
 
+        Given N pericenters at times t[i], i=0...N-1, this function
+        returns a np.array of length N-1, where result[i] is the frequency
+        averaged over [t[i], t[i+1]].
```
Comment (vijayvarma392) [inline gw_eccentricity/eccDefinition.py:1533]: Not clear enough. Point to the equation in the paper, and say what time this average is associated with.

File: gw_eccentricity/eccDefinition.py [hunk 2]
```diff
@@ -1539,17 +1543,17 @@ def get_orbit_averaged_omega22_at_pericenters(self):
         return (np.diff(self.phase22[self.pericenters_location]) /
                 np.diff(self.t[self.pericenters_location]))
 
-    def get_orbit_averaged_omega22_at_apocenters(self):
+    def get_orbit_averaged_omega22_between_apocenters(self):
         """Get orbital average of omega22 between two consecutive apocenters.
 
         The procedure to get the orbital average of omega22 at apocenters is
```
Comment (vijayvarma392) [inline gw_eccentricity/eccDefinition.py:1549]: It still says "at apocenters" here...please check carefully for other instances.

File: gw_eccentricity/eccDefinition.py (hunk 1 above)
Comment (md-arif-shaikh) [reply gw_eccentricity/eccDefinition.py]: Done

File: gw_eccentricity/eccDefinition.py (hunk 2 above)
Comment (md-arif-shaikh) [reply gw_eccentricity/eccDefinition.py]: Done

**Review by vijayvarma392 — APPROVED** [review]

File: gw_eccentricity/eccDefinition.py (hunk 1 above)
Comment (haraldp271) [reply gw_eccentricity/eccDefinition.py]: Thanks @vijayvarma392 for making my suggestion even more precise, and thanks @md-arif-shaikh for implementing it.

**Pull request merged**